### PR TITLE
bump to 1.20.0 node dependencies

### PR DESCRIPTION
**Pull request by paweljakubas**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/da70d67ff662b85e72e1f203defbf5e3028d03cd/snapshots/cardano-1.19.1.yaml
```
Comment (hasufell) [inline stack.yaml:1]: I'd prefer we commit this file to the repository:

* easier to view/grep the dependencies
* unbreaks `stack2cabal` (currently doesn't handle remote urls in `resolver`)
* more static than a url

**Review by rvl — APPROVED** [review]

**Comment by hasufell** [comment]
@paweljakubas did you see my comment?

**Comment by paweljakubas** [comment]
@hasufell oh, sorry, PR approve went to me and I automatically merge it...

Frankly speaking I tried here to comply with practice we use in other repos, like cardano-wallet, cardano-addresses.... Maybe it is good idea to pursue the approach you recommend, first discuss it in our Slack channel, and then adopt in all repos at once to have this level consistency? What do you think? I do not feel expert in those things and always try to comply with pattern used in other repos 

@KtorZ @rvl what are your opinions here?

**Comment by hasufell** [comment]
I don't have a strong opinion. I might just fork stack2cabal anyway, since upstream isn't very active.

**Comment by rvl** [comment]
I suspect that the real objection to resolver URLs is that stack2cabal doesn't work with them and the other reasons are just padding.

**Pull request merged**